packages/sanity/src/structure/panes/document/document-layout/useBundleDeleted.tsx

```diff
@@ -0,0 +1,39 @@
+import {Text, useToast} from '@sanity/ui'
+import {useEffect, useState} from 'react'
+import {Translate, useBundles, usePerspective, useTranslation} from 'sanity'
+
+export const useBundleDeleted = () => {
```

Comment:
I think renaming this to `useBundleDeletedToast` would make it easier to quickly understand what this hook is doing.